spec/grover/middleware_spec.rb

```diff
@@ -431,6 +431,50 @@
         get 'http://www.example.org/test.pdf'
         expect(last_response.body.bytesize).to eq Grover.new('Processed McProcessyface').to_pdf.bytesize
       end
+
+      context 'with root_url specified' do
+        subject(:mock_app) do
+          builder = Rack::Builder.new
+          builder.use described_class, root_url: 'http://example.com/'
+          builder.run downstream
+          builder.to_app
+        end
+
+        it 'calls to the HTML preprocessor with the original HTML and the specified root_url' do
+          expect(Grover::HTMLPreprocessor).to(
+            receive(:process).
+              with('Grover McGroveryface', 'http://example.com/', 'http').
+              and_return('Processed McProcessyface')
+          )
+          get 'http://www.example.org/test.pdf'
+          expect(last_response.body.bytesize).to eq Grover.new('Processed McProcessyface').to_pdf.bytesize
+        end
+      end
+
+      context 'converts relative paths' do
```

Comment:
Rubocop is complaining because it wants the contexts to start with `with`, `without` or `when`.

```
context 'when the response contains relative paths'
```
maybe?